source/_docs/free-https.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Early Access: Free and Automated HTTPS
+earlyaccess: true
+description: Upgrade to Free and Automated HTTPS, powered by Let's Encrypt
+---
+Upgrade your sites's HTTPS and never manage certificates again. Pantheon automatically adds all of your site's domains to a shared certificate and  serves it through our globally distributed content delivery network (CDN). Upgrade to take advantage of performance, security benefits, plus the time and cost of manually purchasing and maintaining certs.
```

Comment:
Can we make it more clear "globally distributed content delivery network" is a new thing?